assets/xml/objects/object_open_obj.xml

```diff
@@ -4,11 +4,11 @@
         <DList Name="object_open_obj_DL_0003E8" Offset="0x3E8" />
         <Texture Name="object_open_obj_Tex_000558" OutName="tex_000558" Format="rgba16" Width="32" Height="64" Offset="0x558" />
         <Collision Name="object_open_obj_Colheader_001640" Offset="0x1640" />
-        <DList Name="object_open_obj_DL_001A60" Offset="0x1A60" />
-        <DList Name="object_open_obj_DL_001B40" Offset="0x1B40" />
-        <Texture Name="object_open_obj_Tex_001CD0" OutName="tex_001CD0" Format="rgba16" Width="32" Height="32" Offset="0x1CD0" />
-        <Texture Name="object_open_obj_Tex_0024D0" OutName="tex_0024D0" Format="ia8" Width="32" Height="64" Offset="0x24D0" />
-        <TextureAnimation Name="object_open_obj_Matanimheader_002CE0" Offset="0x2CE0" />
+        <DList Name="spotlightLeft" Offset="0x1A60" />
+        <DList Name="spotlightRight" Offset="0x1B40" />
+        <Texture Name="spotlightShapesTex" OutName="tex_001CD0" Format="rgba16" Width="32" Height="32" Offset="0x1CD0" />
+        <Texture Name="spotlightFloorTex" OutName="tex_0024D0" Format="ia8" Width="32" Height="64" Offset="0x24D0" />
+        <TextureAnimation Name="spotlightAnimateMat" Offset="0x2CE0" />
```

Comment:
The names are fine except some lack the suffix, you can consult https://github.com/zeldaret/mm/blob/master/tools/ZAPD/docs/zapd_extraction_xml_reference.md for what they should be, and you should also name the textures' OutNames properly with a snake_case version of the texture name.

Reply:
Ah apologies, thanks for the link. I'll get this sorted